packages/ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -609,6 +641,8 @@ export class AngularCompilerPlugin {
     // cleanup if not watching
     compiler.hooks.thisCompilation.tap('angular-compiler', compilation => {
       compilation.hooks.finishModules.tap('angular-compiler', () => {
+        this._warnOnUnusedFiles(compilation);
```

Comment:
Should the list be cleared after the warning? 

I'm not sure how it would work with rebuilds though, because the loader might not request all files again. I'm not sure the unused files are removed from the compilation either.

Reply:
Yeah, that's why I didn't clean it up after each build, because during rebuilds only the changed file will be requested.